pipeline/load.py

```diff
@@ -0,0 +1,59 @@
+"""Script to connect to an MSSQL database and insert plant data"""
+
+
+from os import environ
+
+from dotenv import load_dotenv
+from sqlalchemy import create_engine, sql, Connection
+import pandas as pd
+
+from transform import csv_to_data_frame
+
+
+def create_database_connection() -> Connection:
```

Comment:
Quite nit picky, but maybe think about passing in a `config` argument here (so replace `environ` in this function with `config`, and then pass in environ in the `if __name__ == "__main__"` statement - makes things a bit neater in my head but just a thought.